src/ansys/fluent/core/launcher/standalone_launcher.py

```diff
@@ -246,6 +248,16 @@ def __init__(
             else:
                 self._launch_cmd = self._launch_string
 
+    @property
+    def command(self) -> str:
+        """Return the command for launching Fluent."""
+        return self._launch_string
+
+    @property
+    def subprocess_keywords(self) -> dict[str, Any]:
+        """Return the keyword arguments for launching Fluent via Python subprocess."""
+        return self._kwargs
+
```

Comment:
Where are we going to use this information ? Does it require for the standalone mode only ?